core/src/main/java/org/apache/iceberg/SetStatistics.java

```diff
@@ -32,10 +32,22 @@ public SetStatistics(TableOperations ops) {
     this.ops = ops;
   }
 
+  /**
+   * Set the statistics file for a snapshot.
+   *
+   * @deprecated since 1.8.0, will be removed in 1.9.0 or 2.0.0, use setStatistics(statisticsFile).
```

Comment:
same as above: `use {@link #setStatistics(StatisticsFile)}`

Reply:
fixed in latest commit